themes/dracula_plus.toml

```diff
@@ -16,7 +16,7 @@ yellow = '#FFCB6B'
 blue = '#82AAFF'
 magenta = '#C792EA'
 cyan = '#8BE9FD'
-white = '#F8F9F2'
+white = '#BBBBBB'
```

Comment:
Why did you change this?

Reply:
The Light Gray was not visible in print_colors before this change.

Reply:
The goal is to provide common themes though, regardless of their contrast or "quality". So changing them arbitrarily does not make sense.

Reply:
Ok sure I will revert the change